keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/AdminController.kt

```diff
@@ -36,4 +39,14 @@ class AdminController(
   )
   fun getPausedApplications() =
     actuationPauser.pausedApplications()
+
+  @GetMapping(
+    path = ["/applications"],
+    produces = [MediaType.APPLICATION_JSON_VALUE, APPLICATION_YAML_VALUE]
+  )
+  fun getManagedApplications() =
+    repository.getAll().map {
+      ApplicationSummary(config = it,
```

Comment:
I think it's probably better to create a method on the pauser that returns all paused applications. Then, we can make only one query to the database to get all paused apps. Merging the data here is much cheaper than making a lot of sql calls.

Reply:
It might be worth optimizing the call path by making the repository methods return `Collection<ApplicationSummary>` rather than loading up a bunch of `DeliveryConfigs` and then just plucking a few fields from them. That way you could also include the paused status in a single query.